libs/s25main/ingameWindows/iwStatistics.cpp

```diff
@@ -172,7 +172,8 @@ void iwStatistics::Msg_ButtonClick(const unsigned ctrl_id)
         case 4:
         case 5:
         case 6:
-        case 7: // Spielerportraits
+        case 7:
+        case 8: // Spielerportraits
```

Comment:
maybe do something like `if(ctrl_id - 1 < activePlayers.size())` ... (of course check for the unsigned overflow correctly etc)?
instead of the switch? so that it works for *any* player numbers?

Reply:
yes sounds good